### PR TITLE
feat:post get permissions event listener

### PR DESCRIPTION
**Pull request by kjkooistra-youwe**
Allows permissions manipulation via a custom event listener.
Depends on https://github.com/pimcore/frontend-permission-toolkit/pull/20

### CODE REVIEW

**Comment by kjkooistra-youwe** [comment]
Use case for this event listener: the members bundle provides a custom groups select field, so you're unable to use the provided object relation field.

File: src/Service.php [hunk 1]
```diff
@@ -41,91 +45,66 @@ class Service {
...
             $objectPermissions = $this->getPermissions($permissionObject, $visitedIds);
+            $mergedPermissions = $this->mergeNestedObjectPermissions($mergedPermissions, $permissions, $objectPermissions);
+        }
 
-            foreach($objectPermissions as $key => $value) {
-                if(($permissions[$key] == self::INHERIT || !array_key_exists($key, $permissions)) && $mergedPermissions[$key] != self::ALLOW) {
-                    $mergedPermissions[$key] = $value;
-                }
+        $permissionsEvent = new PermissionsEvent($mergedPermissions, $object, $this);
+        \Pimcore::getEventDispatcher()->dispatch(PermissionsEvent::POST_GET, $permissionsEvent);
```
Comment (fashxp) [inline src/Service.php:78]: should we inject event dispatcher directly via DI and avoid static pimcore call?

Comment (kjkooistra-youwe) [reply src/Service.php:78]: Good point. I'll change that after the refactor is through.

**Comment by fashxp** [comment]
I guess you need to merge master into it

File: src/Service.php (hunk 1 above)
Comment (kjkooistra-youwe) [reply src/Service.php]: Updated and tested

Comment (fashxp) [reply src/Service.php]: thank you very much!

**Pull request merged**